src/app/progress/powercaps.scss

```diff
@@ -0,0 +1,29 @@
+@use '../variables.scss' as *;
+.powercaps {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 1rem;
+  padding-top: 0;
+}
+.powercap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.powercap-label {
+  color: #a1a2a2;
+}
+
+.powercap-value {
+  font-weight: bold;
+  font-size: 1.6rem;
```

Comment:
These look great, but you'll probably need mobile styles.

Reply:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/9866589/226112114-b2f6a4c9-5240-403c-8546-d78affcc2799.png">

How do we feel about this 2x2 layout for mobile?